conpot/protocols/bacnet/bacnet_server.py

```diff
@@ -0,0 +1,427 @@
+# Copyright (C) 2015  Peter Sooky <xsooky00@stud.fit.vubtr.cz>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# Author: Peter Sooky <xsooky00@stud.fit.vubtr.cz>
+# Brno University of Technology, Faculty of Information Technology
+
+import logging
+import re
+import random
+import sys
+import socket
+
+from lxml import etree
+
+from gevent.server import DatagramServer
+
+import bacpypes.object
+
+from bacpypes.app import LocalDeviceObject, BIPSimpleApplication
+from bacpypes.pdu import GlobalBroadcast
+
+from bacpypes.apdu import APDU, apdu_types, confirmed_request_types, complex_ack_types, unconfirmed_request_types, error_types, ConfirmedRequestPDU, UnconfirmedRequestPDU, SimpleAckPDU, ComplexAckPDU, ErrorPDU, RejectPDU, IAmRequest, IHaveRequest, ReadPropertyACK, ConfirmedServiceChoice, UnconfirmedServiceChoice
+
+from bacpypes.basetypes import *
+from bacpypes.primitivedata import *
```

Comment:
Could you do this without the * imports? It's always a bit more complicated to read code this way.